roles/ceph-common/tasks/main.yml

```diff
@@ -90,3 +90,9 @@
 
 - name: include configure_cluster_name.yml
   include: configure_cluster_name.yml
+
+- name: include configure_memory_allocator.yml
+  include: configure_memory_allocator.yml
+  when:
+    - (ceph_origin == 'repository' or ceph_origin == 'distro')
+    - ceph_tcmalloc_max_total_thread_cache is defined
```

Comment:
Also, we need a condition on filestore only `osd_objectstore: filestore`.

Reply:
Does bluestore not use tcmalloc? I thought it did - I havn't messed about with it to confirm - easy enough to fix this, just wanted to confirm.

Reply:
Yup, please leave it only for filestore.